spec/models/bet_option_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe BetOption, type: :model do
```

Comment:
Adding a method, and the spec for that method in a single commit is preferred. 

We track something called code coverage on projects, which is the percentage of your code that gets run during your test suite. We strive to keep all of our Repos at 100% code coverage.

If you add a method in one commit without the associated specs for it, then the code coverage drops for that commit specifically